packages/cli/test/integration-2.test.ts

```diff
@@ -681,6 +681,53 @@ test('vercel env with unknown `VERCEL_ORG_ID` or `VERCEL_PROJECT_ID` should erro
   expect(output.stderr).toContain('Project not found');
 });
 
+test('add a sensitive env var', async () => {
+  const dir = await setupE2EFixture('project-sensitive-env-vars');
+  const projectName = `project-sensitive-env-vars-${
+    Math.random().toString(36).split('.')[1]
+  }`;
+
+  // remove previously linked project if it exists
+  await remove(path.join(dir, '.vercel'));
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏move to cleanup step

Can we clean this up in an `after` or `try`-`finally` instead? I'd prefer to have the state clean in between runs to avoid issues where this kind of cleanup can blind us to misunderstood behavior in the current test if changes are made to it in the future.